go/vt/wrangler/workflow.go

```diff
@@ -715,4 +715,31 @@ func (wr *Wrangler) deleteWorkflowVDiffData(ctx context.Context, tablet *topodat
 	}
 }
 
+// optimizeCopyStateTable rebuilds the copy_state table to ensure the on-disk
+// structures are minimal and optimized and resets the auto-inc value for
+// subsequent inserts.
+// This helps to ensure that the size, storage, and performance related factors
+// for the table remain optimal over time and that we don't ever exhaust the
+// available auto-inc values for the table.
+func (wr *Wrangler) optimizeCopyStateTable(ctx context.Context, tablet *topodatapb.Tablet) {
```

Comment:
Is it worth running this in a goroutine, since `optimize table` might not get prioritized during heavy load?

Reply:
Yeah, that's a good idea. I've done that here: https://github.com/vitessio/vitess/pull/11451/commits/cf259c37b0580bbd122725e0ce706a67647be4f1